### PR TITLE
Move CSharp and VB Desktop code to Portable

### PR DESCRIPTION
**Pull request by jaredpar**
This moves the contents of MS.CA.CSharp.Desktop and MS.CA.VisualBasic.Desktop to the appropriate portable assembly.

### CODE REVIEW

**Comment by jaredpar** [comment]
CC @gafter, @pharring, @Pilchie, @davkean, @jasonmalinowski

File: src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs [hunk 1]
```diff
@@ -262,7 +262,7 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                         try
                         {
                             preferredUILang = new CultureInfo(value);
-                            if ((preferredUILang.CultureTypes & CultureTypes.UserCustomCulture) != 0)
+                            if (CorLightup.Desktop.IsUserCustomCultor(preferredUILang) ?? false)
```
Comment (Pilchie) [inline src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs]: spelling "CustomCultor"?

**Comment by Pilchie** [comment]
:+1:

File: src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs (hunk 1 above)
Comment (jaredpar) [inline src/Compilers/CSharp/Portable/CommandLine/CommandLineParser.cs]: That's shameful, even for me.

**Comment by jaredpar** [comment]
@dotnet-bot  test this please

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Comment by jaredpar** [comment]
@dotnet-bot test this please

**Pull request closed without merging**